docs/source/index.rst

```diff
@@ -1,6 +1,4 @@
-===========================
-Iterative Ensemble Smoother
-===========================
+.. mdinclude:: ../../README.md
```

Comment:
Very cool! I love the re-use of the README.

Reply:
Cool :)